samples/samples/snippets.py

```diff
@@ -1529,7 +1529,11 @@ def update_with_batch_dml(instance_id, database_id):
     )
 
     def update_albums(transaction):
-        row_cts = transaction.batch_update([insert_statement, update_statement])
+        status, row_cts = transaction.batch_update([insert_statement, update_statement])
+
+        # check response status and process error (if occurred)
+        # if status.code != rpc.code_pb2.OK:
+        # ...
```

Comment:
I have concerns about this comment creating incorrect expectations. Yes, you can check the status here and do something different non-OK statuses. However, the error will still be raised by `run_in_transaction` when `commit()` is called unless you raise something else within the transaction logic.

Reply:
Perhaps we should add the check in the sample to show best practices and include comments explaining the peculiarities, e.g.
```
if status.code != OK:
  # Do handling here.
  # Note: the error will still be raised when commit is called by `run_in_transaction`.
  return
```

WDYT, @IlyaFaer?

Reply:
@larkee, yeah, it can be done